.github/workflows/build-test.yml

```diff
@@ -376,47 +315,29 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        jupyter: [raw, local, remote]
+        jupyter: [raw, local]
         python: [nonConda, conda]
         pythonVersion: [3.9]
-        # integration: Tests with VS Code, Python extension & real Jupyter
-        # notebook: Notebook Tests with VS Code, Python extension & real Jupyter
-        # notebookAndWebview: Tests with VS Code (possible Notebooks), WebViews, Python extension & real Jupyter
-        test-suite: [integration, notebook]
         # We're not running CI on macOS for now because it's one less matrix entry to lower the number of runners used,
         # macOS runners are expensive, and we assume that Ubuntu is enough to cover the UNIX case.
         os: [ubuntu-latest]
         # More details on includes/excludes can be found here https://docs.github.com/en/actions/reference/workflow-syntax-for-github-actions#example-including-additional-values-into-combinations
         # Basically with exclude, you can exclude any of the combinations from the result matrix.
         # & with include, you can include additional items to the result matrix.
         exclude:
-          - os: ubuntu-latest # We don't care how users start Remote Jupyter as long as it works.
-            # With this, we're saying from the result matrix, remove the combination that has python 3.9, remote, integration on ubuntu.
-            pythonVersion: '3.9'
-            jupyter: remote
-            test-suite: integration
           - os: ubuntu-latest # We don't care how users start Remote Jupyter as long as it works.
             pythonVersion: '3.9'
             python: conda
             jupyter: remote
-            test-suite: notebook
         include:
-          - os: ubuntu-latest
-            pythonVersion: '3.9'
+          - jupyter: remote
             python: nonConda
-            jupyter: raw
-            test-suite: notebookAndWebview # Test webview only with raw & non-conda.
```

Comment:
No longer required, as we always run webview tests in other suites as well, hence this was just a waste of resources.